Makefile

```diff
@@ -95,7 +96,7 @@ tree-status: ## Verify tree is clean and all changes are committed
 build-cron: build-pubsub build-bq-transfer build-github-server build-webhook build-add-script \
 	  build-validate-script build-update-script
 
-build-targets = generate-docs build-proto build-scorecard build-cron dockerbuild
+build-targets = generate-docs build-proto build-scorecard build-cron ko-build-scorecard ko-build-batch-controller ko-build-batch-worker ko-build-bq-transfer ko-build-webhook ko-build-github-server 
```

Comment:
Create a single `ko-build` target that builds the `ko-build-*` targets.

Reply:
With this approach, I can run individual builds and it makes it easier to build. Also, the plan is to have a few more jobs in  GH actions for each one of these build targets so that it finishes sooner.

Reply:
What I mean is to create a phony target for all these individual targets for better readability. Basically:

```
ko-build: ko-build-scorecard ko-build-controller ...
```

And then use this phony target in `build-target`.

Reply:
I'm updating this comment based on our recent offline conversation: having multiple `ko-build-*` targets specified as `build-target` is redundant and does not improve readability. To help us maintain quality of our code base, please consider updating this line of code to the suggestion above.

Reply:
I am going to accept this. But this approach has a drawback. Our GitHub actions are slow because they all use a single Job. With all these in a single job, it goes sequential but if they were an independent target it would be running parallel which will speed up GH runs.

Reply:
Thanks for updating the code. We can still get parallelism using `make -j`. Using the example above:

```
ko-build: ko-build-scorecard ko-build-controller ...
```

in `.yml` workflow file:

```
make -j <num_parallel_jobs> ko-build
```
 
We can do this in a separate PR. Can submit this as is for now.